src/go/rdctl/cmd/factoryReset.go

```diff
@@ -34,25 +28,18 @@ var removeKubernetesCache bool
 // https://github.com/rancher-sandbox/rancher-desktop/issues/2408)
 
 var factoryResetCmd = &cobra.Command{
-	Use:   "factory-reset",
-	Short: "Clear all the Rancher Desktop state and shut it down.",
+	Use:    "factory-reset",
+	Hidden: true, // Hidden for backwards compatibility, use 'rdctl reset --factory' instead
+	Short:  "Clear all the Rancher Desktop state and shut it down.",
 	Long: `Clear all the Rancher Desktop state and shut it down.
-Use the --remove-kubernetes-cache=BOOLEAN flag to also remove the cached Kubernetes images.`,
+Use the --remove-kubernetes-cache=BOOLEAN flag to also remove the cached Kubernetes images.
+This command is now deprecated in favor of 'rdctl reset --factory'.`,
```

Comment:
There's a [`Deprecated`](https://pkg.go.dev/github.com/spf13/cobra#Command) property that we may use for this purpose, I think?